airbyte-metrics/metrics-lib/src/main/java/io/airbyte/metrics/lib/MetricQueries.java

```diff
@@ -153,44 +153,67 @@ public static List<Pair<JobStatus, Double>> overallJobRuntimeForTerminalJobsInLa
-                      		AND cast(c.schedule::jsonb->'timeUnit' as text) = '"hours"'
-                      	GROUP BY 1
-                      	HAVING count(*) < 24 / cast(c.schedule::jsonb->'units' as integer))
-                      UNION (
-                      SELECT
-                      	c.id,
-                      	count(*) as cnt
-                      FROM connection c
-                      LEFT JOIN Jobs j ON j.scope::uuid = c.id
-                      WHERE
-                      	c.schedule IS NOT null
-                      	AND c.schedule != 'null'
-                      	AND j.created_at > now() - interval '1 hours 1 minutes'
-                      	AND c.status = 'active'
-                      	AND j.config_type = 'sync'
-                      	AND c.updated_at < now() - interval '1 hours 1 minutes'
-                      	AND cast(c.schedule::jsonb->'timeUnit' as text) = '"minutes"'
-                      GROUP BY 1
-                      HAVING count(*) < 60 / cast(c.schedule::jsonb->'units' as integer))) as abnormal_sync_jobs
-                      	""";
-    return ctx.fetch(query).getValues(countField, long.class).get(0).longValue();
+    final var queryForAbnormalSyncInHoursInLastDay = """
+                                                     select count(1) as cnt
+                                                       from
+                                                         (
+                                                         select
+                                                           c.id,
+                                                           count(*) as cnt
+                                                         from
+                                                           connection c
+                                                         left join Jobs j on
+                                                           j.scope::uuid = c.id
+                                                         where
+                                                           c.schedule is not null
+                                                           and c.schedule != 'null'
+                                                           and j.created_at > now() - interval '24 hours 1 minutes'
+                                                           and c.status = 'active'
+                                                           and j.config_type = 'sync'
+                                                           and c.updated_at < now() - interval '24 hours 1 minutes'
+                                                           and cast(c.schedule::jsonb->'timeUnit' as text) = '"hours"'
+                                                         group by 1
+                                                         having count(*) < 24 / cast(c.schedule::jsonb->'units' as integer)) as abnormal_jobs
+                                                     """;
```

Comment:
nit: We can use a string.format method here so the `cnt` name is not repeated. See the other methods in this class for examples.

Reply:
done